settings.gradle

```diff
@@ -6,12 +6,26 @@ pluginManagement {
 }
 
 plugins {
-    id("io.micronaut.build.shared.settings") version "4.2.6"
+    id("io.micronaut.build.shared.settings") version "5.2.1"
 }
 
-rootProject.name = 'nats'
+rootProject.name = 'micronaut-nats'
 
 include 'nats'
 include 'docs-examples:example-groovy'
 include 'docs-examples:example-java'
 include 'docs-examples:example-kotlin'
+
+enableFeaturePreview 'TYPESAFE_PROJECT_ACCESSORS'
+enableFeaturePreview 'VERSION_CATALOGS'

```

Comment:
:x: Using Gradle 7.4 the `VERSION_CATALOGS` feature is no longer preview.

Reply:
I've removed the line.